### ISSUE TITLE
undocumented feature 'panda install .'

### ISSUE THREAD

**Issue by wollmers** [tadzik/panda#288]
Needed a trial that installation from current directory works (like cpanm).

Could have saved me much time, if I had known earlier.

**Comment by tadzik** [comment]
It's noted in the README:

    Alternatively, you can install a package from the local disk by supplying its path:
        panda install ./perl6-Acme-Meow

Can you think of some better wording that would make it more noticable?

**Comment by paultcochrane** [comment]
Have added a note about `panda install .` in 419a6ec.  Does this make the feature sufficiently clear?

**Comment by tadzik** [comment]
419a6ec says "install all dependencies", while it in fact also installs the module itself; or am I not understanding it right?

**Comment by paultcochrane** [comment]
You're right.  That looks like a thinko on my part, sorry.  How does "... install the module and all its dependencies ..." sound?

**Comment by tadzik** [comment]
I would rather just add it as another example in the paragraph before it, which already explains what installing from a directory does :)
